cg/meta/workflow/analysis.py

```diff
@@ -712,7 +712,7 @@ def get_data_analysis_type(self, case_id: str) -> str | None:
         analysis_types: set[str] = {
             link.sample.application_version.application.analysis_type for link in case.links
         }
-        if len(analysis_types) > 1:
+        if len(analysis_types) > 1 and self.workflow != Workflow.MIP_DNA:
```

Comment:
Won't this give an unclear return statement? It just pops the first analysis type in the list but if we have several we might need some extra logic

Reply:
hmm I see... perhaps we should not even go into the function for ip dna but handle the function call based on the workflow?

Reply:
A bit uncertain but it is set in the RarediseaseParameters. Might need to check with some Raredisease developer to know the implications?